frontend/mr-admin-flate/src/pages/avtaler/AvtaleKnapperad.tsx

```diff
@@ -1,14 +1,21 @@
 import { Avtale } from "mulighetsrommet-api-client";
 import { Lenkeknapp } from "../../components/lenkeknapp/Lenkeknapp";
 import styles from "../DetaljerInfo.module.scss";
+import { ViewEndringshistorikk } from "../../components/endringshistorikk/ViewEndringshistorikk";
+import React from "react";
+import { useAvtaleEndringshistorikk } from "../../api/avtaler/useAvtaleEndringshistorikk";
 
 interface Props {
   avtale: Avtale;
 }
 
 export function AvtaleKnapperad({ avtale }: Props) {
+  const { data: historikk } = useAvtaleEndringshistorikk(avtale.id);
+
   return (
     <div className={styles.knapperad}>
+      {historikk && <ViewEndringshistorikk historikk={historikk} />}
```

Comment:
Kan vi bruke ternary istedenfor `&&`, ref: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx